newIDE/app/src/EventsSheet/EventsTree/Renderers/CommentEvent.js

```diff
@@ -108,33 +95,59 @@ export default class CommentEvent extends React.Component<
 
     return (
       <div
-        style={{ ...styles.container, backgroundColor: `#${color}` }}
         className={classNames({
           [largeSelectableArea]: true,
           [largeSelectedArea]: this.props.selected,
         })}
-        ref={container => (this._container = container)}
+        style={{
+          ...styles.container,
+          backgroundColor: `#${backgroundColor}`,
+          minHeight: 35,
+        }}
+        onClick={this.edit}
       >
-        {!this.state.editing ? (
-          <p
+        {this.state.editing ? (
+          <TextField
+            multiLine
+            margin="none"
+            ref={textField => (this._textField = textField)}
+            value={commentEvent.getComment()}
+            hintText={t`<Enter comment>`}
+            onBlur={this.endEditing}
+            onChange={this.onEvent}
+            style={{ ...styles.commentSpan }}
```

Comment:
It's not necessary to use ... here, it's creating a new object in memory for nothing:
`style={styles.commentSpan}`

Reply:
I've fixed that :)